HotelManagerment/com.spring.sample/WebContent/html/sessions/billForRent/BillForRent.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
HotelManagerment/com.spring.sample/WebContent/html/sessions/billForRent/BillForRent.html
đổi tên nào
HotelManagerment/com.spring.sample/WebContent/html/sessions/bill-for-rends/index.html